### PR TITLE
Add username/pwd auth support

### PR DESCRIPTION
**Pull request by ZIMkaRU**
This PR adds corresponding changes to add BFX auth token support to framework mode. Basic changes:
  - Adds `invalidateAuthToken` and `generateToken` endpoints
  - Adds ability to send emitBfxUnamePwdAuthRequiredToOne event by WebSockets in cases if the auth token is expired for the framework mode
  - Adds ability to use dynamic getter to fetch refreshed auth token from a session in the framework mode

### CODE REVIEW

File: workers/loc.api/helpers/get-rest.js [hunk 1]
```diff
@@ -141,11 +141,19 @@ module.exports = (conf) => {
     const {
       apiKey = '',
       apiSecret = '',
-      authToken = '',
-      ip = ''
```
Comment (ezewer) [inline workers/loc.api/helpers/get-rest.js:145]: No ip needed for authToken?

**Review by ezewer — COMMENTED** [review]
Small question, rest looks fine

File: workers/loc.api/helpers/get-rest.js (hunk 1 above)
Comment (ZIMkaRU) [reply workers/loc.api/helpers/get-rest.js:145]: yes, it works without `ip`
and looks like `ip` was removed from libs
https://github.com/bitfinexcom/bitfinex-api-node/blob/master/index.js#L22
https://github.com/bitfinexcom/bfx-api-node-rest/blob/master/lib/rest2.js#L85

**Review by ezewer — APPROVED** [review]
looks good to me!

**Pull request merged**